packages/oui-slideshow/src/slideshow.component.js

```diff
@@ -9,6 +9,7 @@ export default {
         onDismiss: "&",
         loading: "<?",
         loop: "<?",
-        theme: "@?"
+        theme: "@?",
+        onNext: "&"
```

Comment:
Please use it in an example ;)

Reply:
Yes, with a note of what is returned with this event (eg. `index`)